### DISCUSSION TITLE
How to avoid extra distribution of zlib.dll

### DISCUSSION THREAD

**Discussion by harry-xi** [scala-native/scala-native#4011] [Q&A] [answered]
On Windows, is it possible for me to avoid supplying an additional zlib.dll in order to make a scala native program that depends on zlib work?

**Comment by He-Pin** [comment]
I found its a totally painful too, I ecounter this wen try to build sjsonnet on Windows with Scala native.

**Comment by harry-xi** [comment, marked as answer]
just follow the workflow for test scala native in this repo, let
```
$env:SCALANATIVE_INCLUDE_DIRS="<vcpkg-static-dir (etc D:\vcpkg\installed\x64-windows-static)>\include"
$env:SCALANATIVE_LIB_DIRS="<vcpkg-static-dir>\lib"
```
then install the zlib with `static` linkage use vcpkg
